src/js/pages/jobs/JobDetailPage.js

```diff
@@ -143,22 +143,27 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
   }
 
   getActionButtons() {
+    let job = MetronomeStore.getJob(this.props.params.id);
+
     let dropdownItems = [
       {
         className: 'hidden',
         html: 'More',
         id: JobActionItem.MORE
       },
       {
-        html: 'Disable Schedule',
-        id: JobActionItem.SUSPEND
-      },
-      {
         html: 'Destroy',
         id: JobActionItem.DESTROY
       }
     ];
 
+    if (job.getSchedules().length > 0) {
+      dropdownItems.splice(1, 0, {
```

Comment:
Whenever we want/need to change the oder we need to also adjust the `splice` call which could result in errors. If we just push the buttons in the correct order it's pretty easy as one only need to move the calls.